Sources/TootSDK/TootClient/TootClient+TimeLine.swift

```diff
@@ -52,6 +52,15 @@ public extension TootClient {
         return try await getPosts(req, pageInfo, limit)
     }
     
+    /// Retrieves the user's local timeline
+    /// - Parameters:
+    ///   - pageInfo: a PageInfo struct that tells the API how to page the response, typically with a minId set of the highest id you last saw
+    ///   - limit: Maximum number of results to return (defaults to 20 on Mastodon with a max of 40)
+    /// - Returns: a PagedResult containing the posts retrieved
+    func getLocalTimeline(_ pageInfo: PagedInfo? = nil, limit: Int? = nil) async throws -> PagedResult<[Post]> {
```

Comment:
How is this method different from the one above? `query` is already optional in the other one too. Won't there be conflicts if someone passes no parameters?

Reply:
A previous PR introduced something that caused these signatures to appear with leading nil at the call site:
```swift
return try await getFederatedTimeline(nil, pageInfo, limit: limit)
```

So, this change adds an overload so they remain available without the leading `nil`.

Reply:
@kkostov I incorrectly assumed that it would be possible to call original function without nil as first parameter. I see now that is not possible because swift gives error about incorrect type.

About the error/warning part, I didn't see how swift would decide which function will be called since call site matched both. But looks like it has some rules that I don't know.